libs/platform/src/lib/components/table/table.component.scss

```
@@ -65,31 +108,104 @@
 
         max-width: 200px;
         min-width: 200px;
-    }
 
-    .fdp-table__cell--selection {
-        min-width: 2.8rem;
-        max-width: 2.8rem;
+        @include fd-table-nested-cell-padding(20, true);
 
-        padding: 0 !important;
+        &.fdp-table__cell--tree-no-expand {
+            @include fd-table-nested-cell-padding(20, true, true);
+        }
+
+        .fdp-table__cell--expand {
+            cursor: pointer;
+            display: inline-flex;
+            justify-content: center;
+            align-items: center;
+            font-size: 0.75rem;
+            color: var(--sapContent_IconColor, #0854A0);
```

Please, add color with plain hex _(without variable)_ for IE
 
```suggestion
            color: #0854A0;
            color: var(--sapContent_IconColor, #0854A0);
```